### ISSUE TITLE
Is this compatible to the "new" Sync (1.5)?

### ISSUE THREAD

**Issue by luckydonald** [owncloud/mozilla_sync#142]
Also known as Sync-1.5, [here is their guide](https://docs.services.mozilla.com/howtos/run-sync-1.5.html)

From the document there I guess it is not, as the custom 
```services.sync.tokenServerURI``` in ```about:config``` is something like 
```http://sync.example.com/token/1.0/sync/1.5```, but the current documentation says
```http://YOURCLOUDDOMAIN/remote.php/mozilla_sync/ ```

**Comment by RealRancor** [comment]
Nope, doesn't work as discussed here https://github.com/owncloud/mozilla_sync/issues/33
